main.go

```diff
@@ -147,7 +152,7 @@ func main() {
 
 				// Create cache
 				cache := job.NewMemoryJobCache(db)
-				cache.Start(DefaultPersistEvery)
+				cache.Start(time.Duration(c.Int("persist-every")) * time.Second)
```

Comment:
Let's use the built-in duration string parsing - https://golang.org/pkg/time/#ParseDuration

This allows for `4h` instead of `14400`

Would also mean making it a `StringFlag` with a default of `5m`.

Lastly lets remove the `DefaultPersistEvery = 5 * time.Second` at the top of this file.